dacbench/benchmarks/toysgd_benchmark.py

```diff
@@ -0,0 +1,136 @@
+import os
+
+import numpy as np
+import pandas as pd
+from gym import spaces
+
+from dacbench.abstract_benchmark import AbstractBenchmark, objdict
+import importlib
+import dacbench.envs.toysgd
+importlib.reload(dacbench.envs.toysgd)
+
+HISTORY_LENGTH = 40
```

Comment:
Is the history length used somewhere I don't see right now?

Reply:
oops forgot to delete, not used right now